RecoTracker/LSTCore/standalone/README.md

```diff
@@ -0,0 +1,291 @@
+#Before writing the following xml file, check that libsdl_cpu.so and libsdl_gpu.so can be found under the ../../../TrackLooper/SDL/ folder.
+cat <<EOF >lst_headers.xml
+<tool name="lst_headers" version="1.0">
+  <client>
+    <environment name="LSTBASE" default="$PWD/../../../TrackLooper"/>
+    <environment name="INCLUDE" default="\$LSTBASE"/>
+  </client>
+  <runtime name="LST_BASE" value="\$LSTBASE"/>
+</tool>
+EOF
+cat <<EOF >lst_cpu.xml
+<tool name="lst_cpu" version="1.0">
+  <client>
+    <environment name="LSTBASE" default="$PWD/../../../TrackLooper"/>
+    <environment name="LIBDIR" default="\$LSTBASE/SDL"/>
+    <environment name="INCLUDE" default="\$LSTBASE"/>
+  </client>
+  <runtime name="LST_BASE" value="\$LSTBASE"/>
+  <lib name="sdl_cpu"/>
+</tool>
+EOF
+cat <<EOF >lst_cuda.xml
+<tool name="lst_cuda" version="1.0">
+  <client>
+    <environment name="LSTBASE" default="$PWD/../../../TrackLooper"/>
+    <environment name="LIBDIR" default="\$LSTBASE/SDL"/>
+    <environment name="INCLUDE" default="\$LSTBASE"/>
+  </client>
+  <runtime name="LST_BASE" value="\$LSTBASE"/>
+  <lib name="sdl_cuda"/>
+</tool>
+EOF
+scram setup lst_headers.xml
+scram setup lst_cpu.xml
+scram setup lst_cuda.xml
+cmsenv
+git cms-checkdeps -a -A
+scram b -j 12
+```
+
+### Run the LST reconstruction in CMSSW
+A simple test configuration of the LST reconstruction can be run with the command:
+```bash
+cmsRun RecoTracker/LST/test/LSTAlpakaTester.py
+```
+
+For a more complete workflow, one can run a modified version of the 21034.1 workflow.
+To get the commands of this workflow, one can run:
+```bash
+runTheMatrix.py -w upgrade -n -e -l 21034.1
+```
+
+For convenience, the workflow has been run for 100 events and the output is stored here:
+```bash
+/data2/segmentlinking/CMSSW_14_1_0_pre0/step2_21034.1_100Events.root
+```
+
+For enabling the LST reconstruction in the CMSSW tracking workflow, a modified step3 needs to be run.
+This is based on the step3 command of the 21034.1 workflow with the following changes:
+   - Remove the `--pileup_input` and `--pileup` flags.
+   - The number of threads and streams for the job can be optionally controlled by the `--nThreads` and `--nStreams` command line options respectively (`1` ends up being the actual default value for both, and more info can be found by running `cmsDriver.py --help`).
+   - Add at the end of the command: `--procModifiers gpu,trackingLST,trackingIters01 --no_exec`
+
+Run the command and modify the output configuration file with the following:
+   - If want to run a cpu version, remove the ```gpu``` in the line defining the `process` object:
+     ```python
+     process = cms.Process('RECO',...,gpu,...)
+     ```
+   - Add the following lines below the part where the import of the standard configurations happens:
+     ```python
+     process.load('Configuration.StandardSequences.Accelerators_cff')
+     process.load("HeterogeneousCore.AlpakaCore.ProcessAcceleratorAlpaka_cfi")
```

Comment:
`Accelerators_cff` already imports `ProcessAcceleratorAlpaka_cfi`, so the latter is redundant
```suggestion
     process.load('Configuration.StandardSequences.Accelerators_cff')
```